pyproject.toml

```diff
@@ -65,18 +65,15 @@ dev = [
 convert = [
     "pynxtools[apm,em,mpes,xps,stm,xrd,ellips]",
 ]
+mpes = [
+    "pynxtools-mpes",
+]
 apm = [
     "pynxtools-apm",
 ]
 em = [
     "pynxtools-em",
 ]
-mpes = [
-    "pynxtools-mpes",
-]
-xps = [
```

Comment:
Please enable the xps plugin again here.

Reply:
Noted!